### ISSUE TITLE
Permit literals

### ISSUE THREAD

**Issue by lpil** [michaeljones/gleam-templates#20]
```
Unexpected identifier: False)

error: 
   ┌─ ./src/todomvc/web/templates/home.gleamx:68:49
   │
68 │           {[ item_template.render_builder(item, False) ]}
   │                                                 ^^^^^^
```

Perhaps the parser could be extra permissive and consume anything until it finds `}}` or `]}`

**Comment by michaeljones** [comment]
Hopefully should be fixed in 0.11.0. 

I realise you might have found other approaches to this by now.

**Comment by lpil** [comment]
Thank you!
